.github/workflows/pgvector.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         python-version: ["3.9", "3.10", "3.11"]
     services:
       pgvector:
-        image: ankane/pgvector:latest
+        image: pgvector/pgvector:pg17
```

Comment:
unrelated: I noticed that we were using a very old Docker image for testing and I updated it